dashboard/test/factories/pd_factories.rb

```diff
@@ -820,10 +820,9 @@
     pay_fee Pd::Application::TeacherApplication.options[:pay_fee].first
     plan_to_teach Pd::Application::TeacherApplication.options[:plan_to_teach].first
     completing_on_behalf_of_someone_else 'No'
-    cs_how_many_minutes 45
-    cs_how_many_days_per_week 5
-    cs_how_many_weeks_per_year 20
-    cs_total_course_hours 75
+    cs_how_many_minutes '45'
+    cs_how_many_days_per_week '5'
+    cs_how_many_weeks_per_year '20'
```

Comment:
This change was to make the form_data coming in consistent––strings instead of ints, `cs_total_course_hours` calculated in `on_successful_create`. It also conveniently helps with my test